conf/messages

```diff
@@ -132,6 +132,8 @@ paymentsHistory.vatErrorCorrectionCreditChargeTitle =  Error correction repaymen
 paymentsHistory.vatErrorCorrectionChargeDescription = for correcting your {0} return
 paymentsHistory.vatRepaymentSupplementTitle = Late repayment compensation from HMRC
 paymentsHistory.vatRepaymentSupplementDescription = we took too long to repay your {0} return
+paymentsHistory.AADefaultInterestDescription = interest charged on additional tax assessed for the period {0} to {0}
```

Comment:
I think this should be
`interest charged on additional tax assessed for the period {0}`
because `displayDateRange` is given to the message which sorts out the start and end date for you